### DISCUSSION TITLE
Option for an Open Shop Catalog

### DISCUSSION THREAD

**Discussion by SumeetMoray** [Open-network-for-digital-commerce/ONDC-Protocol-Specs#3] [General]
Hello everyone at ONDC ?

Creating a product catalog and keeping product prices up to date on multiple e-commerce platforms can be really painful. Painful both for the platform and the shop owner also. 

This can be greatly simplified if we have an open-source and Single shop catalog which can serve as a single point for updating prices and SKUs by a shop and This catalog can be shared and used by all e-commerce platforms ?

Any progress in this regard ? Any thoughts, ideas and Plans ?

**Comment by venkatramanm** [comment]
I am working on this. 
Take a look at https://opencatalog.readthedocs.io

You are most welcome to work together on this initiative. Ondc is in talks with brands like pepsi and hul . Initial discussions are very promising. 

I also have developed a  web app to do self-service by brands (poc).

**Reply by SumeetMoray** [reply]
hi ... thats great ... is this project a part of ONDC ? ... or you are developing it independently ... i am interested to work together on this project !

Have you started coding ? ... or its still in concept stage ... I have one open source e-commerce project .... we can use the code to create an open catalog !

Let me know what you think !

**Comment by adityar-r** [comment]
hi, is there any update on this? I also want to contribute to this code base.

**Reply by venkatramanm** [reply]
https://github.com/venkatramanm/opencatalog 

Has the proof of concept code.

**Comment by utkarsh4321** [comment]
Hii I m new to this. But I want to connect my store with ondc as individual . Can anyone tell how I can run the seller app locally and call the api.

**Comment by utkarsh4321** [comment]
And also want to know after submitting network participants form what I will get
